docker/base/Dockerfile

```diff
@@ -59,12 +59,12 @@ RUN curl -sL https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.3/install.sh | b
   nvm install $NODE_VERSION
 
 RUN add-apt-repository universe && apt-get update && \
-  apt-get install -y python3.10 python3-pip && \
+  apt-get install -y python3 python3-pip python3-setuptools && \
   update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1 && \
   update-alternatives --install /usr/bin/python python /usr/bin/python3 1
 
 RUN groupadd -r blessuser && useradd --uid ${BLESS_USER_ID} -r -g blessuser -G audio,video blessuser && \
   mkdir -p /home/blessuser/Downloads && \
   chown -R blessuser:blessuser /home/blessuser
 
-RUN npm i --production=false
+RUN npm clean-install --production=false
```

Comment:
Isn't the `--production=false` deprecated? Are we really wanting to install dev/optional dependencies here? Should this be using the `--include` flag?